ui/app/pages/permissions-connect/permissions-connect.component.js

```diff
@@ -122,11 +152,9 @@ export default class PermissionConnect extends Component {
             request={permissionsRequest || {}}
             approvePermissionsRequest={ (requestId, accounts) => {
               approvePermissionsRequest(requestId, accounts)
-              this.redirectFlow(true)
```

Comment:
Nit: There's another `redirectFlow(false)` that could probably be removed, up on line 146.

Though... do we want to remove these lines? I don't see why we'd want to wait for a re-render before starting the redirect flow. I'm guessing this was to prevent it from being called twice? You could instead check `state.page` in `onComponentUpdate`, and bail if it's `null` already.

Reply:
1b68fb882